hathor/p2p/protocol.py

```diff
@@ -155,11 +150,9 @@ def __init__(self, network: str, my_peer: PeerId, connections: Optional['Connect
         self.aborting = False
 
         self.use_ssl: bool = use_ssl
-        self.enable_sync_v1: bool = enable_sync_v1
-        self.enable_sync_v2: bool = enable_sync_v2
 
         # Protocol version is initially unset
-        self.protocol_version = ProtocolVersion.UNSET
+        self.sync_version = None
```

Comment:
Where do we set this attribute to the one selected?

Reply:
On `hathor/p2p/states/hello.py` line `129`.